src/Apps/Artwork/Components/PrivateArtwork/PrivateArtworkAboutArtist.tsx

```diff
@@ -37,48 +65,101 @@ export const PrivateArtworkAboutArtist: React.FC<PrivateArtworkAboutArtistProps>
     artwork
   )
 
-  // FIXME: Remove (typechecker)
+  const followsCount =
+    Number(formatFollowerCount(data.artist?.counts?.follows)) ?? 0
 
-  if (!data.displayArtistBio) {
-    return null
-  }
+  const biographyBlurb =
+    data.artist?.partnerBiographyBlurb?.text ??
+    data.artist?.biographyBlurb?.text ??
+    ""
```

Comment:
Assuming private artworks will take greater advantage of the partner bio blurb in CMS, so prioritize that

Reply:
Unless I'm misunderstanding the code, I don't know if it is this simple? There are multiple choices here, firstly starting with whether the artist bio should be shown at all or not (`displayArtistBio`)
![Screenshot 2024-04-17 at 14 58 50](https://github.com/artsy/force/assets/6280410/fe38e0ef-e829-4e84-8546-3e68a2a86fdd)

Then there is a second choice which is either use the Artsy provided Artist bio, or show a partner provided Artist Bio (`biographyBlurb` vs `partnerBiographyBlurb`)
![Screenshot 2024-04-17 at 14 55 48](https://github.com/artsy/force/assets/6280410/da88022a-0307-416f-99b7-8c5373f40f4e)